UpdatedImageProcessing/Classifiers/classify_tester.py

```diff
@@ -0,0 +1,156 @@
+import sys
+from os import walk
+import numpy as np
+import json
+import timeit
+
+from classify_color import ColorClassifier
```

Comment:
Use a relative import.

Reply:
We talked this over. Relative imports don't work in the Classifiers folder.

Reply:
Yes they do. You just need to test outside of the package. This needs to be fixed.